README.md

```diff
@@ -4,7 +4,7 @@
 ![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/TheAlgorithms/C-Plus-Plus?color=green&style=flat-square)
 
 ### All algorithms implemented in C++ (for education)
-The implementations are for learning purpose. They may be less efficient than the implementation in the standard library.
+The implementations are for learning purposes, wherein some may be less efficient than their corresponding implementations in the standard library.
```

Comment:
The implementations are for learning purposes.  They may be less efficient than corresponding implementations in the standard library.

Reply:
`wherein` is not helpful.